src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/AssessmentService.kt

```diff
@@ -743,8 +743,7 @@ class AssessmentService(
     currentAssessment.reallocatedAt = OffsetDateTime.now()
     assessmentRepository.save(currentAssessment)
 
-    // Make the timestamp precision less precise, so we don't have any issues with microsecond resolution in tests
-    val dateTimeNow = OffsetDateTime.now().withNano(0)
+    val dateTimeNow = OffsetDateTime.now()
```

Comment:
Interesting! Have you got a link to docs on the `TimeExtensions.roundNanosToMillisToAccountForLossOfPrecisionInPostgres` function you mention in the commit message?

Reply:
It's one of ours - https://github.com/ministryofjustice/hmpps-approved-premises-api/blob/main/src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/util/TimeExtensions.kt

The name is a bit verbose, any proposals to shorten it would be appreciated :)

Reply:
👍 ah, I see, thks!